drivers/watchdog/wdt_nrfx.c

```diff
@@ -124,6 +90,15 @@ static int wdt_nrf_install_timeout(struct device *dev,
 	}
 
 	if (m_allocated_channels == 0) {
+		/* According to relevant Product Specifications, watchdogs
+		 * in all nRF chips can use reload values (determining
+		 * the timeout) from range 0xF-0xFFFFFFFF given in 32768 Hz
+		 * clock ticks or 0x1-0x07CFFFFF in milliseconds. Check if
```

Comment:
Specifications don't mention the range in milliseconds.
So maybe: "... ticks. This makes the allowed range of 0x1-0x07CFFFFF in milliseconds. Check if the provided value is within this range."